### ISSUE TITLE
Github authentication sometimes stops working

### ISSUE THREAD

**Issue by pchote** [warwick-one-metre/dashboard#21]
Restarting the dashboard service temporarily fixes it.
Deleting local cookies and clearing the `dashboard_sessions` table do not help.

The uwsgi log contains
```
Failed to clean expired session data with error
(0, '')
Failed to query local session data with error
(0, '')
Failed to query GitHub API with error
(0, '')
```

**Comment by pchote** [comment]
It looks like the database connection is timing out when left long enough without a query.  There is a `db.ping()` method that we can call to reestablish the connection before trying to do real work.

**Comment by pchote** [comment]
Fixed by 52132fdaf383941972d19d559a00638f89a3b489.
